boards/shared/Src/Io/Io_SharedErrorTable.c

```diff
@@ -0,0 +1,33 @@
+#include "App_SharedErrorTable.h"
+#include "App_CanMsgs.h"
+#include "Io_SharedErrorTable.h"
+
+static void Io_ProcessFsmCriticalErrorMsg(
+    struct ErrorTable *          error_table,
+    struct CanMsgs_fsm_errors_t *data)
+{
+    // TODO: Fill this out once I'm sure about other parts of the design
```

Comment:
TODO? This design seems fairly mature to me now.

Reply:
Fixed, added for all boards

Reply:
Is it added for all boards? I don't see it.

Reply:
Refresh one more time? It's there i just double checked

Reply:
What files is this in? I see no calls to this function anywhere.

Reply:
Oh I thought I misunderstood what you were asking. I thought you meant if I wrote all the `Io_Process{Board}NonCriticalErrorMsg` function definitions. 

I wanted to call the function in another PR where I add in the callback functions for CAN TX/RX interface. This PR will just be focused on building the error table itself. How does that sound?

Reply:
Oh that's totally fine, just making sure we're on the same page.